perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -480,6 +480,10 @@ def _pywb_static_dir_location():
 # http://remote-webrecorder-host:8089
 WR_API = 'http://nginx/api/v1'
 
+# WR Credentials for the public user that stores all public collections
+WR_PERMA_USER = 'perma'
+WR_PERMA_PASSWORD = os.environ.get('WR_PERMA_PASSWORD', 'TestTest123')
```

Comment:
Fun, hidden feature: all Perma settings can be overwritten from env vars without going though this (courtesy of https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/settings/utils/environmental_settings.py). So here, you can just do `WR_PERMA_PASSWORD = 'TestTest123'` or whatever, and users can a) customize in a private `settings.py` file per usual for Perma secrets or b) set the env var `DJANGO__WR_PERMA_PASSWORD` and it will get picked up automatically.